src/DotNetStatus/Common/AnalyzerInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Fx.Portability
+{
+    public class AnalyzerInfo
+    {
+        public string AnalyzerName { get; set; }
+        public string Id { get; set; }
+        public ICollection<CompatibilityRangeAttribute> Attributes { get; set; }
+        public bool IsCompatibilityDiagnostic { get; set; }
+
+        public int IdNumber
+        {
+            get
+            {
+                var matches = Regex.Match(Id, @"(\d+)");
```

Comment:
Looks like this has some startup performance issues https://blogs.msdn.microsoft.com/bclteam/2004/11/12/regular-expression-performance-david-gutierrez/